src/live/src/user-to-user/live-service-facade.js

```diff
@@ -0,0 +1,58 @@
+import PubNub from 'pubnub';
+
+import Client from '../../../client';
+import { User } from '../../../entity';
+import { PubNubListener } from './pubnub-listener';
+import { getLiveService } from './live-service';
+
+
+function _getLiveService() {
+  const client = Client.sharedInstance();
+  return getLiveService(client);
+}
+
+function register() {
```

Comment:
Definitely shouldn't expose both. If you guys prefer it on the `User` class, will move it there. But not the logic, just the call. I much rather the logic is kept inside `LiveService`. Does that sound good to you?